src/Pies.js

```diff
@@ -264,18 +272,9 @@ const Pies = React.createClass({
         const _startAngle = circularScale(0);
         this.color = helpers.colorFunc(colors);
 
-        const coords = helpers.getCoords(position || '', layerWidth, layerHeight, outerRadius * 2, outerRadius * 2);
-
-        const transform = 'translate3d(' + (coords.x + outerRadius) + 'px,' + (coords.y + outerRadius) + 'px,0px)';
-        const chartStyle = _.defaults({
-            transform,
-            WebkitTransform: transform,
-            MozTransform: transform
-        }, style);
-
         const halfPadAngle = props.padAngle / 2 || 0;
 
-        return <g className={className} style={chartStyle} opacity={opacity}>
+        return <g className={className} style={style} opacity={opacity}>
```

Comment:
I'd assume because I needed to move the `transform` here for proper display, but perhaps it could still live in the `render` method (rather than `renderArc`). Feel free to try moving it back and seeing what it does.